### PR TITLE
Removed old code

### PR DESCRIPTION
**Pull request by antonvas**
This code is not required but pulled during npm install. This is a fix

### CODE REVIEW

**Comment by amyboyd** [comment]
How is there nearly 4k lines unused? Are these files from the original library we forked?

**Comment by antonvas** [comment]
Yes these are original files. Which is not used

**Comment by amyboyd** [comment]
+1

File: src/js/classes/crop-area-circle.js [hunk 1]
```diff
@@ -1,154 +0,0 @@
-(function () {
```
Comment (BernardoSilva) [inline src/js/classes/crop-area-circle.js:1]: @antonvas if we remove this how can we support crop as a circle shape?
Did you rebuild this functionality somewhere else?

Comment (antonvas) [inline src/js/classes/crop-area-circle.js:1]: There no such functionality now. Because it is not required in pm2 for now :)

Comment (BernardoSilva) [inline src/js/classes/crop-area-circle.js:1]: But the functionality was already there, why remove it? :s

Comment (antonvas) [inline src/js/classes/crop-area-circle.js:1]: Because editor was completely rewritten. And it using different approach

Comment (BernardoSilva) [inline src/js/classes/crop-area-circle.js:1]: Then to keep it consistent:
- [ ] Project description is wrong.
  
  > Image Editor directive for AngularJS. Enables to crop a **circle** or a square out of an image.
- [ ] First screenshot example is incorrect: https://github.com/partnermarketing/pm-image-editor#screenshots
- [ ] Options are not correct, need to be updated to the ones we have now.
  https://github.com/partnermarketing/pm-image-editor#options
- [ ] References to circle i here as well:
  https://github.com/partnermarketing/pm-image-editor#result-image
- [ ] Non existing config: https://github.com/partnermarketing/pm-image-editor#area-type

Comment (antonvas) [inline src/js/classes/crop-area-circle.js:1]: Fixed

**Comment by BernardoSilva** [comment]
:+1: Much better, thanks @antonvas

**Pull request merged**